evaluator.py

```diff
@@ -126,6 +135,15 @@ def final(**available_fileds):
         return final
     return middle
 
+@default
+@inputs(COSTS)
+def total_costs(costs):
```

Comment:
Can it just be "costs"?
We don't say "total returns" because when it comes to returns, of course it should be total ones.